### DISCUSSION TITLE
Project Still Alive?

### DISCUSSION THREAD

**Discussion by mountainbiker54** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#68] [General]
Does anyone know if this project is still alive. I ordered a unit from the website over a month ago. I've tried contacting the seller multiple times with no response at all. If anyone knows how to contact their sales staff please post.

Thanks

**Comment by CircuitSetup** [comment]
Absolutely!

Sorry I haven't gotten back to you yet, as I've fallen a bit behind on emails.

The next batch of meters is being manufactured, and the last batch is currently being shipped. Always look on the product pages for backorder updates: https://circuitsetup.us/product/expandable-6-channel-esp32-energy-meter/

**Comment by rmvandek** [comment]
Also I am wondering if the project is still alive! I ordered 3 board's and a lot of otherthings on the second of December. Hoping that I am in the backorder of the 15th of December. Send different emails but get no reaction other then my first one that my order is recieved and will be proccesed. Please update the status and send me at least a reaction. If you cant deliver please let us know, and if you can't deliver in the near future please refund my payment. 
Thanks

**Comment by CircuitSetup** [comment]
I'm currently working through the massive amount of backorders, and up to orders placed at the end of November (with some exceptions due to other items being backordered).

Sorry for the delay.
